src/components/headers/formHeader/FormHeaderButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { GestureResponderEvent, Pressable, Text } from 'react-native'
+
+interface FormHeaderButtonProps {
+  onPress?: ((event: GestureResponderEvent) => void) | null | undefined
```

Comment:
can this just be:
`(event: GestureResponderEvent) => void)`
as it's been marked as optional already? (`onPress?:`)

Reply:
👍